openbb_terminal/stocks/options/tradier_model.py

```diff
@@ -142,21 +142,78 @@ def get_full_option_chain(symbol: str) -> pd.DataFrame:
     """
 
     expirations = option_expirations(symbol)
+    options_dfs: pd.DataFrame = []
 
-    options = pd.DataFrame()
+    for expiry in expirations:
+        options_dfs.append(get_option_chains(symbol, expiry))
 
-    for date in expirations:
-        temp = get_option_chains(symbol, date)
-        calls = temp[temp.option_type == "call"][option_cols].rename(
-            columns=option_col_map
-        )
-        puts = temp[temp.option_type == "put"][option_cols].rename(
-            columns=option_col_map
-        )
-        temp = pd.merge(calls, puts, how="outer", on="strike", suffixes=("_c", "_p"))
-        temp["expiration"] = date
+    options_df = pd.concat(options_dfs)
```

Comment:
Note you are saying options_dfs is a dataframe then defining a list.

You can simply define options_df = pd.DataFrame() and then concat in the loop

df = pd.DataFrame()
for expiry in expirations:
   df = pd.concat([df,get_option_chains(symbol,expiry)])